toplevel/role/role.go

```diff
@@ -19,13 +19,17 @@ import (
 type entry struct {
 	Name        string                 `yaml:"name"`
 	Type        string                 `yaml:"type"`
-	Mount       string                 `yaml:"mount"`
+	Mount       authMount              `yaml:"mount"`
 	Instance    vault.Instance         `yaml:"instance"`
 	OutputPath  string                 `yaml:"output_path"`
 	Options     map[string]interface{} `yaml:"options"`
 	Description string                 `yaml:"description"`
 }
 
+type authMount struct {
+	Path string `yaml:"_path"`
```

Comment:
Took me a minute to understand what was going on here, but checking through https://github.com/app-sre/qontract-schemas/blob/9d6509ffdb3a4428d1f873a5a1c753e556768939/graphql-schemas/schema.yml#L2951, I see that this is a field returned in GQL queries

Reply:
correct. Good highlight though, I'll add a comment referencing where that's being mapped from